components/SearchField.js

```diff
@@ -0,0 +1,173 @@
+import styled from "styled-components";
+import Image from "next/image";
+import { useState } from "react";
+
+export default function SearchField({
+  onSortComplexityStartHigh,
+  onSortComplexityStartLow,
+  onSortDuration,
+  query,
+  handleSearch,
+  onFilterProjects,
+  onResetFilters,
+}) {
+  const [isOpenSort, setIsOpenSort] = useState(false);
+  const [isOpenFilter, setIsOpenFilter] = useState(false);
+
+  function handleChange(event) {
```

Comment:
You could pick a more specific name here than handleChange, so in a later stage of your project you can easily identify the functionality behind your function.

Reply:
Thanks Henrik, good point that I will change immediately.